lms/djangoapps/certificates/views/webview.py

```diff
@@ -254,20 +254,7 @@ def _update_course_context(request, context, course, platform_name):
     course_number = course.display_coursenumber if course.display_coursenumber else course.number
     context['course_number'] = course_number
     context['is_integrity_signature_enabled_for_course'] = is_integrity_signature_enabled(course.location.course_key)
-    if context['organization_long_name']:
-        # Translators:  This text represents the description of course
-        context['accomplishment_copy_course_description'] = _('a course of study offered by {partner_short_name}, '
-                                                              'an online learning initiative of '
-                                                              '{partner_long_name}.').format(
-            partner_short_name=context['organization_short_name'],
-            partner_long_name=context['organization_long_name'],
-            platform_name=platform_name)
-    else:
-        # Translators:  This text represents the description of course
-        context['accomplishment_copy_course_description'] = _('a course of study offered by '
-                                                              '{partner_short_name}.').format(
-            partner_short_name=context['organization_short_name'],
-            platform_name=platform_name)
+    context['accomplishment_copy_course_description'] = context['certificate_data'].get('course_description', '')
```

Comment:
@pkulkark actually, setting the default value to empty means that we take away the default text. I think this is something we wouldn't like to do as it could cause confusion -- why isn't the `a course of study offered by {partner_short_name}...` text is there.

I would suggest having the course description as an override. If the value is set, we don't care about the default text, otherwise we default to the removed condition.

Reply:
Good point. Made the changes.